app/marketing/views.py

```diff
@@ -845,8 +863,11 @@ def leaderboard(request, key=''):
     if next_update and next_update < timezone.now():
         next_update = timezone.now() + timezone.timedelta(days=1)
 
+    dualList= zip(items[0:limit],flags)
```

Comment:
please use snake case for variable names: https://en.wikipedia.org/wiki/Snake_case

Reply:
whoops. fixed!